test/test_hashlesstilecoder.jl

```diff
@@ -0,0 +1,25 @@
+
+using MinimalRLCore
```

Comment:
Because of removing the export we should add another using to import the HashTilecoders into the Main namespace.

Reply:
looks like this isn't necessary